runner/runners/log_uploader.go

```diff
@@ -0,0 +1,31 @@
+package runners
+
+// LogUploader provides functionality to upload logs to permanent storage, used by invoker to upload task logs
+type LogUploader interface {
+	UploadLog(blobId string, filepath string, cancelCh chan struct{}) (string, error)
```

Comment:
Was going to say we tend to prefer `blobID` to `blobId` as far as naming, but we have occurrences of the latter around anyway.